content/applications/general/apps_modules.rst

```diff
@@ -2,73 +2,84 @@
 Apps and modules
 ================
 
-You can :ref:`install <general/install>`, :ref:`upgrade <general/upgrade>` and :ref:`uninstall
-<general/uninstall>` all apps and modules from the :menuselection:`Apps` dashboard.
+:ref:`Install <general/install>`, :ref:`upgrade <general/upgrade>` and :ref:`uninstall
+<general/uninstall>` any needed apps and modules from the :menuselection:`Apps` dashboard.
 
-By default, an *Apps* filter is applied. If you want to search for modules, click on
-*Filters* and select *Extra*.
+By default, an *Apps* filter is applied. To search for
+modules, click on *Filters* and select *Extra*.
```

Comment:
UI text should be formatted with a :guilabel:.

Also the instruction to click "Filter" is not correct, since "Filters" is a section.

I suggest rewriting to:
```suggestion
By default, an :guilabel:`Apps` filter is applied. To search for modules too, select
:guilabel:`Extra` from the :icon:`fa-filter` :guilabel:`Filters`.
```